recipes-debian/npth/npth_debian.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "New GNU Portable Threads library"
```

Comment:
@cheat-sc  could you add following fixed comment please.
```
# base recipe: 
# base branch: 
# base commit: 
```

Reply:
I'm sorry I forgot to add them. I have added them.

Reply:
I'm sorry I forgot to add them. I have added them.

Reply:
thanks